modules/sxiv/testbeds/nsxiv.nix

```
@@ -0,0 +1,19 @@
+{ lib, pkgs, ... }:
+
+let
+  package = pkgs.nsxiv;
+  image = pkgs.fetchurl {
+    url = "https://raw.githubusercontent.com/NixOS/nixos-artwork/4ad062cee62116f6055e2876e9638e7bb399d219/logo/nixos.svg.png";
+    hash = "sha256-9+OfqfP5LmubdTcwBkS/AnOX4wZI2tKHLu5nhi43xcc=";
+  };
+  script = pkgs.writeShellScript "load" ''
+    # Xresources isn't loaded by default, so we force it
+    ${lib.getExe pkgs.xorg.xrdb} ~/.Xresources
+    ${lib.getExe package} ${image}
+  '';
+in
+{
+  stylix.testbed.ui.command.text = script.outPath;
```

No need to wrap with `pkgs.writeShellScript`, since `stylix.testbed.ui.command.text` already does exactly that:

https://github.com/danth/stylix/blob/ce5fcf851d6546a24e5b5b17ee7051b4b384be79/stylix/testbed.nix#L142

You probably want something like:

```suggestion
  stylix.testbed.ui.command.text = let
    image = /* ... */;
  in ''
    # Xresources isn't loaded by default, so we force it.
    ${lib.getExe pkgs.xorg.xrdb} ~/.Xresources
    ${lib.getExe package} ${image}
  '';
```